linkage/demo_notebooks/2022_10_21a_prl_splink_test_part_1.ipynb

```diff
@@ -0,0 +1,3242 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    training_blocking_rules = ["l.last_name = r.last_name", 'l.zipcode = r.zipcode', 'l.age + 10 = r.age']
</pre><p>These seem like very permissive blocking rules for training, no? Using matching-last-name or matching-zipcode as a proxy for true matches will probably significantly underestimate m for the other parameters trained in that training session.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>